### DISCUSSION TITLE
Jobset config help

### DISCUSSION THREAD

**Discussion by MerkyLive** [ericgrandt/TotalEconomy#392] [Ideas]
Need assistance with my Jobset Config with farmer and Fisher. I keep getting job farmer has the nonexistent set and job fisher has the nonexistent set. I've placed the ids with the mod from Pams harvest.

here is the Jobset config if anyone has any idea that be much help!! 
https://gist.github.com/MerkyLive/5f880f63faa67ebecdb8b875a82b6b9e

**Comment by ericgrandt** [comment]
Hi @MerkyLive, starting on line 381 you have to surround the names of the fish with quotes (") since there is a space in the name.

**Reply by MerkyLive** [reply]
I've tried it with the quotes and also did 1 word(ex trout not "raw trout") and still not working. Should i use the id that is on Pams harvest wiki? or should the names of the fish work?

**Reply by ericgrandt** [reply]
Could you show your updated jobsets file? Is it still saying both the farmer and fisher have non existent sets? And what are the names of the sets that don't exist?

**Reply by MerkyLive** [reply]
i went back to just having fish as 1 word its not showing the error in console but players are also not earning exp or money with the new fish that where added. Same thing also with farmers but farmers aren't receiving exp or money for anything.

https://gist.github.com/MerkyLive/15ce00b667e73ace8f10e9a0c6436e67

**Reply by MerkyLive** [reply]
So I found out farmers only works when you break cops from villages for the first time once you replant them it won't count

**Reply by ericgrandt** [reply]
If the new fish aren't giving exp/money then the id is incorrect. You'll have to find the correct id for the mod you're using. In regards to the farmer job, I think this might have something to do with `growthTrait=age`. Not 100% sure how it works though. I'm assuming when you break the crop after placing it you're waiting until it's fully grown?
